replica/database.cc

```diff
@@ -915,7 +915,7 @@ void database::before_schema_keyspace_init() {
     _uses_schema_commitlog = true;
 
     db::commitlog::config c;
-    c.commit_log_location = _cfg.commitlog_directory();
+    c.commit_log_location = _cfg.schema_commitlog_directory();
```

Comment:
Should we default it do be in a sub-directory of commitlog_directory()? This way existing setups will just work even if users forget to update scylla.yaml. There's a risk that the schema commitlog will end up in the wrong disk, together with data directory.

Reply:
good point, thanks